Tests/LibWeb/Text/expected/HTML/HTMLLinkElement-appends-and-removes-from-ShadowRoot-styleSheets.txt

```diff
@@ -0,0 +1,13 @@
+== before appending
+is linkEl.sheet null? true
+- document stylesheets (length = 0)
+- shadow root stylesheets (length = 0)
+== link loaded
+is linkEl.sheet null? false
+- document stylesheets (length = 0)
+- shadow root stylesheets (length = 1)
+-- shadow root stylesheet href: file:///Users/lukewilde/Repositories/ladybird/Tests/LibWeb/Text/input/valid.css
```

Comment:
This is the line CI doesn't like! Probably not a good idea to print this href as it'll be different on every machine.

(We really should stop resolving URLs in the CSS parser. :sweat_smile:)